frontend/src/data/trees.tsx

```diff
@@ -7,35 +7,35 @@ export function treeDemoData() {
       species: 'Quercus robur',
       number: '100123',
       hasSensor: true,
-      status: WateringStatus.bad,
+      status: EntitiesTreeClusterWateringStatus.TreeClusterWateringStatusGood,
     },
     {
       id: 1,
       species: 'Crataegus monogyna',
       number: '100123',
       hasSensor: true,
-      status: WateringStatus.moderate,
+      status: EntitiesTreeClusterWateringStatus.TreeClusterWateringStatusModerate,
     },
     {
       id: 2,
       species: 'Crataegus monogyna',
       number: '100123',
       hasSensor: true,
-      status: WateringStatus.good,
+      status: EntitiesTreeClusterWateringStatus.TreeClusterWateringStatusGood,
     },
     {
       id: 3,
       species: 'Quercus robur',
       number: '100123',
       hasSensor: false,
-      status: WateringStatus.unknown,
+      status: EntitiesTreeClusterWateringStatus.TreeClusterWateringStatusUnknown,
     },
     {
       id: 4,
       species: 'Quercus robur',
       number: '100123',
       hasSensor: false,
-      status: WateringStatus.unknown,
+      status: EntitiesTreeClusterWateringStatus.TreeClusterWateringStatusUnknown,
```

Comment:
I think that's a misnomer, but that's my mistake in the backend. Perhaps the name will be changed to TreeClusterWateringStatus.TreeClusterWateringStatusUnknown.